### PR TITLE
changes configmap file name in boot mission. Fixes RHOARDOC-1280

### PR DESCRIPTION
**Pull request by rhoads-zach**
@inoxx03 @Ladicek @gytis let me know what you think.

relates to https://github.com/snowdrop/spring-boot-configmap-booster/commit/16885a75fcbc82a92d4a9b54aede8336568a77de

### CODE REVIEW

File: docs/topics/proc_deploying-the-configmap-booster-using-the-oc-cli-client.adoc [hunk 1]
```diff
@@ -110,10 +110,10 @@ $ oc create configmap app-config --from-file=greeting-service/src/main/etc/appli
...
 $ oc get configmap app-config -o yaml
 
-apiVersion: v1
-data:
-  application.properties: |-
-      greeting.message=Hello %s from a ConfigMap!
+application.yml: |
+   # This properties file should be used to initialise a ConfigMap
+   greeting:
+     message: "Hello %s from a ConfigMap!"
```
[the remark this answers is not in the document]
Comment (rhoads-zach) [reply docs/topics/proc_deploying-the-configmap-booster-using-the-oc-cli-client.adoc]: Sorry, lost those bits in the copy/paste. re-added.

File: docs/topics/configmap-mission-basic-interaction-spring-boot.adoc [hunk 2]
```diff
@@ -17,7 +17,7 @@ $ curl http://{app-name}-{project-name}.{os-route-hostname}/api/greeting
 $ oc edit configmap app-config
 ----
 +
-Change the value for the `greeting.message` key to `Bonjour!` and save the file. After you save this, the changes will be propagated to your OpenShift instance.
+Change the value for the `message` key to `Bonjour!` and save the file. After you save this, the changes will be propagated to your OpenShift instance.
```
[the remark this answers is not in the document]
Comment (rhoads-zach) [reply docs/topics/configmap-mission-basic-interaction-spring-boot.adoc]: fixed

**Comment by rhoads-zach** [comment]
@Ladicek thank you for the feedback. amended the PR

**Comment by Ladicek** [comment]
Thanks @rhoads-zach! Hopefully you found all the places to change, I didn't check :-)

**Comment by rhoads-zach** [comment]
Thanks @Ladicek i did a find for the old path in the spring boot booster. This covers guide and readme.

**Pull request merged**